swag_client/backend.py

```diff
@@ -15,11 +15,19 @@
 logger = logging.getLogger(__name__)
 
 
-def validate(item, namespace='accounts', version=2):
-    """Validate item against version schema."""
+def validate(item, namespace='accounts', version=2, context={}):
```

Comment:
Don't use a mutable type in a `kwarg` (`context`).

Reply:
Thanks - committed a fix.